### PR TITLE
Create 2018-12-02 WCA Board Meeting.md

### PR DESCRIPTION
**Pull request by AlbertoPdRF**

### CODE REVIEW

File: documents/minutes/2018-12-02 WCA Board Meeting.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Christopher Wright <br>
+Olivér Perge <br>
+Robert Burton <br>
+</div>
+
+<b class="text-center">WCA MOTION AMENDMENTS</b>
+
+The Board agreed upon the amendments to the Motions that will be proposed to WCA Staff and External Staff for review. The review period will last two weeks. Then, the proposed amendments will be voted on by all WCA Voting members and finally by the Board of Directors. If both votes are successful the new Motions will come into effect on January 1, 2019.
+
+<b class="text-center"WCA DUES</b>
```
Comment (operge) [inline documents/minutes/2018-12-02 WCA Board Meeting.md]: > missing before WCA DUES

File: documents/minutes/2018-12-02 WCA Board Meeting.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+</div>
+
+On December 2, 2018 at 9:13 a.m. PST, the Board of Directors of World Cube Association (the “Board”) met for a regular meeting via Google Hangouts. The following directors were present at the meeting:
+
+<div class="text-center">
+Alberto Pérez de Rada Fiol <br>
+Christopher Wright <br>
+Olivér Perge <br>
+Robert Burton <br>
+</div>
```
Comment (operge) [inline documents/minutes/2018-12-02 WCA Board Meeting.md]: I would use an empty line before the names and the title.

Comment (AlbertoPdRF) [reply documents/minutes/2018-12-02 WCA Board Meeting.md]: There's one! (Line 15)

File: documents/minutes/2018-12-02 WCA Board Meeting.md (hunk 1 above)
Comment (AlbertoPdRF) [reply documents/minutes/2018-12-02 WCA Board Meeting.md]: Nice catch! I just added it.

**Pull request merged**